packages/apps/plugins/plugin-thread/src/components/CommentContainer.tsx

```diff
@@ -133,7 +141,8 @@ export const CommentContainer = ({
             extensions={extensions}
           />
           <ThreadFooter activity={activity}>{t('activity message')}</ThreadFooter>
-          <AnchoredOverflow.Anchor />
+          {/* NOTE(thure): This can’t also be the `overflow-anchor` because `ScrollArea` injects an interceding node that contains this necessary ref’d element. */}
```

Comment:
is this related to the issue about ensuring scrolling a thread into view scrolls to the text input as opposed to the top of the thread?

Reply:
Is there still an outstanding issue? I believe we resolved #5401 if that’s what you’re referring to.

Speaking more generally, `ScrollArea` and `overflow-anchor` seem to be incompatible because `ScrollArea` adds an interceding node. I haven’t found a non-JS workaround just yet.

Reply:
Yeah, that's the issue I was referring to. Was just wondering if this was related or not.